Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/Inputs/SwitchInput.swift

```diff
@@ -73,6 +73,9 @@ struct SwitchInput: View {
             .onChange(isOn) { isOn in
                 element.updateValue(isOn ? input.onValue.code : input.offValue.code)
                 model.evaluateExpressions()
+            }
+            .onTapGesture {
+                isOn.toggle()
```

Comment:
`SwitchInput` needs to use a tap gesture to know when to set the focused element.

If we set the focusedElement during the onChange callback, the element is unexpectedly focused and scrolled to when a form first opens.

Reply:
Using a tap gesture on a switch (and setting the value bound to the switch in the handler) is extremely unusual. Is the unexpected focus and scroll behavior due to this:

https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/blob/49a1b26be1ac612f3b78f0b273062dad60bd153f/Sources/ArcGISToolkit/Components/FeatureFormView/FeatureFormView.swift#L114-L118

If so, the unintended behavior should be addressed at the source rather than hacking a switch like this.

Reply:
Sorry, I meant unexpected as in unexpected _to the user_. They'll open a form and it'll be scrolled down to a switch and they won't know why.

What we want to avoid is the element setting itself as the focused element when this line runs for the first time:
https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/blob/bcdf01233e1f98e14c068819bb756ba8555091e2/Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/Inputs/SwitchInput.swift#L73

By using `onTapGesture` the element isn't set as the focused element in the model until the user taps on it.

My change here reverted #1048 to the previous implementation, but I will look for a better solution.

Reply:
@philium I've committed another approach. This is ready for review again.